libs/sync/event.go

```diff
@@ -141,7 +152,7 @@ func (e *EventSyncComplete) String() string {
 func newEventComplete(seq int, put, delete []string, dryRun bool) Event {
 	return &EventSyncComplete{
 		EventBase:    newEventBase(seq, EventTypeComplete, dryRun),
-		EventChanges: &EventChanges{Put: put, Delete: delete},
```

Comment:
Seems like what you need is `EventChanges{Put: slices.Sorted(put), Delete: slices.Sorted(delete)}` ?

Reply:
changed to use slices.Sorted(slices.Values()) in the follow-up commit, but kept the `newEventChanges` method to match the style of the rest of the code in this file